2020-11-06 22:40:54,331 ERROR: Exception on /user/RoRo [GET] [in /Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/Users/rohit/Downloads/VIP/app/routes.py", line 30, in user
    return render_template('user.html', user=user)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: user.html
2020-11-06 23:14:31,823 ERROR: Exception on /user/RoRo [GET] [in /Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/Users/rohit/Downloads/VIP/app/routes.py", line 30, in user
    return render_template('user.html', user=user)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: user.html
2020-11-06 23:24:58,212 ERROR: Exception on /user/RoRo [GET] [in /Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/Users/rohit/Downloads/VIP/app/routes.py", line 30, in user
    return render_template('user.html', user=user)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/rohit/Downloads/VIP/env/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: user.html
